builtin/verify-tag.c

```diff
@@ -51,8 +56,20 @@ int cmd_verify_tag(int argc, const char **argv, const char *prefix)
 		const char *name = argv[i++];
 		if (get_sha1(name, sha1))
 			had_error = !!error("tag '%s' not found.", name);
-		else if (gpg_verify_tag(sha1, name, flags))
-			had_error = 1;
+		else {
+			if (fmt_pretty) {
+				verify_ref_format(fmt_pretty);
```

Comment:
Thanks for uncommenting! This is actually very important because it does not only verify but also parses out used atoms.

I don't like to put it here though, because we don't have to check the same format string for every gpg-verified tag iterated over, but only once, e.g. in [`cmd_tag`](https://github.com/lukpueh/git/blob/master/builtin/tag.c#L321).